### PR TITLE
Adding a PR template

### PR DESCRIPTION
**Pull request by kodebae**
## Adding a PR template
* The purpose of this PR is to add a template for future PRs.
* The suggested change solves the issue of not having a template ready for members.
* I am requesting the following feedback from @klmansel.

### Type of Change
* Update to the documentation.

### How Has This Been Tested?
The change can not be tested in a traditional sense.

### CODE REVIEW

**Comment by klmansel** [comment]
@kodebae can you resolve the merge conflicts and rename the PR?

File: README.md [hunk 1]
```diff
@@ -64,6 +64,33 @@ There are two ways to use git to interact with our repository: command-line Git
...
+<p>Screenshot 1</p>
+<img src="#" alt="your alt description goes here" width="55%" height="40%">
+
+<p>Screenshot 2</p>
+<img src="#" alt="your alt description goes here" width="55%" height="40%">
+
+<p>Screenshot 3</p>
+<img src="#" alt="your alt description goes here" width="55%" height="40%">
+-->
+
```
Comment (klmansel) [inline README.md:93]: The PR template is normally used automatically when a PR is opened. Here is the [process](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository). We don't need to add it to the README, but a link to the template in the contributing file is ok

File: README.md [hunk 2]
```diff
@@ -113,6 +140,8 @@ Project Link: [https://github.com/WomenWhoCode/WWCodeGreenville](https://github.
 
 - [WWCode Greenville Notion](https://wwcode.notion.site/Women-Who-Code-Greenville-02e4990c6f814dccb92cf24da404b4b8)
 
+- [WWCode Greenville Online Event Recordings](../WWCodeGreenville/WWCode_Event_Recordings/EventRecordings.md)
```
Comment (klmansel) [inline README.md:143]: same question as the other PR - I'm not sure why these changes are in both

**Review by klmansel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (klmansel) [reply README.md]: @kodebae making sure you saw this comment 😁

Comment (kodebae) [reply README.md]: I've never created a PR template for anyone before. Let me look at what you linked and get back to you.

**Comment by klmansel** [comment]
> @kodebae can you resolve the merge conflicts and rename the PR?

@kodebae I changed the name for you. You will also want to reference the Issue # in the description so it closes automatically, but I got you!

File: README.md (hunk 1 above)
Comment (kodebae) [reply README.md]: @klmansel I believe I may have fixed everything. Let me know if there is anything else I still need to do and I'd be happy to fix it.
